src/connector/src/parser/util.rs

```diff
@@ -109,3 +114,38 @@ pub(super) async fn bytes_from_url(url: &Url, config: Option<&AwsAuthProps>) ->
         )))),
     }
 }
+
+pub fn extreact_timestamp_from_meta(meta: &SourceMeta) -> Option<Datum> {
```

Comment:
```suggestion
pub fn extreact_kafka_timestamp_from_meta(meta: &SourceMeta) -> Option<Datum> {
```

Reply:
Wait, it might be better to add a method to `KafkaMeta`, like `get_timestamp(&self) -> Option<Datum>`

Reply:
done